frontend/src/pages/dashboard.tsx

```diff
@@ -64,39 +43,37 @@ const Dashboard: NextPage = (): JSX.Element => {
                 <h2 className="ml-6 font-semibold">Dashboard</h2>
                 <div className="flex items-center">
                   <div className="flex-shrink-0">
-                    <Link href={`/profile/${author?.id}`}>
+                    <Link href={`/profile/${data?.data?.user?.id}`}>
```

Comment:
why is this `data.data`?
```suggestion
                    <Link href={`/profile/${data.data.user?.id}`}>
```

Reply:
Can you destructure the inner `data` instead of accessing it from the swr `data` constant? 

`const { data } = data`

then access user id from there